ui/mirage/handlers/clients.js

```diff
@@ -663,6 +664,7 @@ const handleMockQuery = (queryStartTimestamp, monthlyData) => {
     do {
       i++;
       let timestamp = formatRFC3339(sub(startDateByMonth, { months: i }));
+      // this is probably not what the empty object looks like but waiting to hear back from backend
```

Comment:
Can we add a "TODO" here so we make sure to come back and clean it up once we do hear back from backend?

Reply:
Great idea!